packages/localizations/src/sv-SE.ts

```diff
@@ -511,51 +514,56 @@ export const svSE: LocalizationResource = {
-    form_identifier_exists__phone_number: 'Telefonnumret finns redan.',
-    form_identifier_exists__username: 'Användarnamnet finns redan.',
-    form_identifier_not_found: 'Identifieraren hittades inte.',
-    form_param_format_invalid: 'Formatet är ogiltigt.',
+    form_code_incorrect: undefined,
+    form_identifier_exists: undefined,
+    form_identifier_exists__email_address: 'Denna e-postadress är taget. Vänligen prova ett annat.',
+    form_identifier_exists__phone_number: 'Detta telefonnummer är taget. Vänligen prova ett annat.',
+    form_identifier_exists__username: 'Detta användarnamn är taget. Vänligen prova ett annat.',
+    form_identifier_not_found: undefined,
+    form_param_format_invalid: undefined,
     form_param_format_invalid__email_address: 'E-postadressen måste vara en giltig e-postadress.',
     form_param_format_invalid__phone_number: 'Telefonnumret måste vara i ett giltigt internationellt format.',
     form_param_max_length_exceeded__first_name: 'Förnamnet får inte överskrida 256 tecken.',
     form_param_max_length_exceeded__last_name: 'Efternamnet får inte överskrida 256 tecken.',
     form_param_max_length_exceeded__name: 'Namnet får inte överskrida 256 tecken.',
-    form_param_nil: 'Parametern får inte vara tom.',
+    form_param_nil: undefined,
     form_param_value_invalid: undefined,
-    form_password_incorrect: 'Lösenordet är felaktigt.',
-    form_password_length_too_short: 'Lösenordet är för kort.',
+    form_password_incorrect: undefined,
+    form_password_length_too_short: undefined,
     form_password_not_strong_enough: 'Ditt lösenord är inte tillräckligt starkt.',
     form_password_pwned: 'Lösenordet har läckt i tidigare dataintrång.',
     form_password_pwned__sign_in: 'Lösenordet har läckt, vänligen logga in för att ändra det.',
     form_password_size_in_bytes_exceeded:
       'Ditt lösenord har överskridit det maximala antalet tillåtna bytes, vänligen förkorta det eller ta bort några specialtecken.',
     form_password_validation_failed: 'Felaktigt lösenord',
-    form_username_invalid_character: 'Användarnamnet innehåller ogiltiga tecken.',
-    form_username_invalid_length: 'Användarnamnets längd är ogiltig.',
+    form_username_invalid_character: undefined,
+    form_username_invalid_length: undefined,
     identification_deletion_failed: 'Du kan inte ta bort din sista identifiering.',
-    not_allowed_access: 'Åtkomst nekad.',
-    organization_domain_blocked: undefined,
-    organization_domain_common: undefined,
-    organization_membership_quota_exceeded: undefined,
-    organization_minimum_permissions_needed: undefined,
+    not_allowed_access: undefined,
+    organization_domain_blocked: 'Domänen är blockerad.',
+    organization_domain_common: 'Domänen är vanlig.',
+    organization_membership_quota_exceeded: 'Medlemskapet är fullt.',
+    organization_minimum_permissions_needed: 'Du måste ha tillräckligt med behörigheter.',
```

Comment:
Same here, I must have been blind or something, the ones before was correct and can be reverted if I made them undefined

Reply:
Fixed in latest commit